### ISSUE TITLE
error in process sentinel: Symbol’s value as variable is void: whisper--stdout-buffer

### ISSUE THREAD

**Issue by liar666** [natrys/whisper.el#17]
Hi,

I just upgraded my DoomEmacs installation.

No I get `error in process sentinel: Symbol’s value as variable is void: whisper--stdout-buffer` when I call `whisper-run`.

Any idea where does this come from? And how to solve it?

**Comment by natrys** [comment]
Really hard to say from this without knowing more. Can you replicate the error if:
- whisper.el is updated to latest commit
- emacs is restarted

if so, please turn on backtrace with `(setq debug-on-error t)` and post the backtrace shown upon error. Also please post your whisper.el config.

**Comment by liar666** [comment]
Hi,

Thanks for the quick response.

It seems to be working now that I've downloaded the latest `whisper.el` from git.

FWIW, I had/have the following lines in my `config.org`:
```
 (use-package! whisper
  :load-path
    "~/.doom.d/packages/whisper.el"
```
And I don't know why "~/.doom.d/packages/whisper.el" had been removed. 
I don't know either which version was used then (probably the default from DoomEmacs' repositories or an old compiled cache?).
But seems to be solve now.
